dancepose/analyze_dance.py

```
@@ -0,0 +1,298 @@
+        """Загружает позы из poses.jsonl"""
+        poses = []
+        with open(poses_file, 'r') as f:
+            for line in f:
+                poses.append(json.loads(line))
+        return poses
+    
+    def analyze_from_poses(
+        self, 
+        poses_file: str,
+        model_path: str = "dance_classifier/best_model_10pct.pth",
+        video_path: str = None
```

Ruff (0.14.5)

PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)